### ISSUE TITLE
Crash in Subversion 3.0.1 Release

### ISSUE THREAD

**Issue by VK3SIR** [KI7MT/jtsdk-dotnet-core#3]
Greg,

I am getting a crash in Subversion when trying to compile the JTDX-fork of wsjtx, and there is no apparent reason why this should occur as the current development release of wsjtx (1.9.2 ?) compiles and deploys cleanly.

i.e.  JTSDK Version        : v3.0.1  Git Tag        : b9216de

The JTSDK 3.0.1 deploys cleanly according to instructions posted; I am using directions provided in the message "JTBuild-cm - WSJT-X Test Build Script" post #435 from 7/29 as the base set of instructions to compile wsjtx with the exception that I build the code into a package. The crash occurs at multiple points before final compilation can occur.

I receive a dialog box stating "The application was unable to start correctly (0xc000007b)"... When I click to terminate the application compilation stops. If I ignore this box it will grind along and stop finally around the 85% mark heading nowhere.

Note that I would ask first in the JTDX groups - but contributions are not appreciated as significant ethical and security concerns that I am raising are being discounted as being negative (i.e. geoblocking). 

73

Steve I
VK3SIR

**Comment by KI7MT** [comment]
Hi Steve,

I know nothing about JTDX. I've never built it (ever), and know nothing about its construct. All I know is that it uses the real WSJT-X source code, which my scripts are designed to support. I would guess the JTDX folks have issues with with their build process and they will need to address that.

As a side note, the official WSJT-X sources are being worked through Git now, not Subversion. Subversion was added to JTSDK-Tools just as a general-use tool for developers, not specifically for WSJT-X work.

73's
Greg, KI7MT

**Comment by VK3SIR** [comment]
Greg,

Thanks. Was suspected. 

My issue relates to subversion finding the dll's it needs I suspect …. 

I have tried regular "pacman" steps to perform an upgrade to version 10.1 without success - but it remains at version 1.9.7 after completion. Can you please guide me how to perform an upgrade just in case I am missing something?

Thanks

Steve I

**Comment by KI7MT** [comment]
Hi Steve,

I think there was an update not long ago for MSYS2 and the pacman database from
the MSYS2 developers. That would require a full update I think. However, I did
not add Subversion too the list of packages to install as its not needed by any
of the scripts I've written for MSYS2. The following pacman query indicates that 1.10.x
version is available:

pacman -Ss subversion

which returns:
msys/subversion 1.10.0-2 (VCS)
    A Modern Concurrent Version Control System

To Upgrade your MSYS2 Instance:

* Open JTSDK-Tools, and type: msys2
* In the MSYS2 Console, type: pacman -Syuu
* Close using the X button, then repeat the update
* packan -Syuu

The first command updates the packman database, and the second updates packages
to their latest release versions.

In general, if you're using JTSDK-Tool Environment (Windows Console), and trying
to build an svn checkout, you should not need Subversion from MSYS2, as 1.9.7
is included with the core tool set. As far as I can determine, 1.9.7 is working
as expected in JTSDK-Tools Windows Console.

You could also be hitting a conflict between the MSYS2 and Windows versions of
Subversion. I've not tested SVN having both Subversion(s) installed and on the
Windows %PATH%. To test for this conflict, you need to disable Unix tools in
JTSDK-Tools:

* Open JTSDK-Tools Environment
* Type: jtconfig -e unix
* Close then re-open JTSDK-Tools Environment

At that point, unix tools are no longer on the %PATH% for JTSDK-Tools
Environment, and should eliminate any potential conflicts between the two packages.

Basically, MSYS2 or the Windows console will pick up whichever version it finds
first on the Windows %PATH%. If the 1.9.7.x version is found first, that is what
will be used.

73's
Greg, KI7MT
